src/App.tsx

```diff
@@ -1,61 +1,109 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [hasSelectError, setSelectError] = useState(false);
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setTitleError] = useState(false);
+  const [todos, setTodos] = useState(todosFromServer);
+
+  const cleanTitle = (str: string) => {
+    return str.replaceAll(/[^\w\d\sа-яїієґА-ЯЇІЄҐ]/g, '');
+  };
+
+  const handleSelection = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedUser(+event.target.value);
+    setSelectError(false);
+  };
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(cleanTitle(event.target.value));
+    setTitleError(false);
+  };
+
+  const onSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setTitleError(!title);
+    setSelectError(!selectedUser);
+
+    if (!title || !selectedUser) {
```

Comment:
i can create the todo only with spaces, let's fix that
<img width="221" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/a79445e3-73cd-4961-ace0-d34363688b1c">